api/api-shared-v10/src/main/java/com/thoughtworks/go/apiv10/admin/shared/representers/stages/ApprovalRepresenter.java

```diff
@@ -33,9 +33,7 @@ public static void toJSON(OutputWriter jsonWriter, Approval approval) {
         }
 
         jsonWriter.add("type", approval.getType());
-        if (approval.isAllowOnlyOnSuccess()) {
-            jsonWriter.add(Approval.ALLOW_ONLY_ON_SUCCESS, approval.isAllowOnlyOnSuccess());
-        }
+        jsonWriter.add(Approval.ALLOW_ONLY_ON_SUCCESS, approval.isAllowOnlyOnSuccess());
```

Comment:
why is this change required?

Reply:
IMO, we should always serialize the boolean values. This will keep the payload consistent. I don't see point why we should not remove the property when value is set to false.